src/synced_services/toggl/toggl_synced_service.ts

```diff
@@ -337,6 +338,32 @@ export class TogglTrackSyncedService implements SyncedService {
     return entries;
   }
 
+  async replaceTimeEntryDescription(timeEntry: ServiceTimeEntryObject, tagName: number | string) {
+    let timeEntryFromApi = await this.getTimeEntryById(timeEntry.id, new Date(new Date().setDate(new Date().getDate() - 180)));
```

Comment:
Tento riadok sa mi vyrazne nepaci, ak sa neda optimalizovat 3x pouzitie new Date, tak navrhujem preniest do funkcie, napr. getDate180DaysBefore

Reply:
Ono ja bych zde nejradeji vubec datum neresil. Metoda `getTimeEntryById` ma mit volitelny parametr `start`, ktery je ale nakonec povinny. Mozna by do budoucna stalo za to udelat parametr povinny, nebo korektne nepovinny.

Reply:
Suhlas, sprav ho nepovinny aj teraz, ak to nie je vela prace.